src/docs/schema_element_deprecation_guide.md

```diff
@@ -0,0 +1,33 @@
+# Deprecating NMDC schema elements
+
+By deprecating, rather than deleting, schema elements in the NMDC model, we ensure backward compatibility, extend user 
+trust, and provide a valuable historical record, aiding in the understanding of the model's evolution. Therefore, 
+'deleting' a schema element in NMDC is instead, a two-step (and two-release) deprecation process.
```

Comment:
Replace all references of "NMDC" in this text with "GSC/MIxS"

Reply:
https://github.com/GenomicsStandardsConsortium/mixs/commit/4c1f364f963932c4471c9760b787dbfb1e102a84